packages/core/src/controllers/BlockchainApiController.ts

```diff
@@ -242,6 +245,39 @@ export const BlockchainApiController = {
     })
   },
 
+  async lookupEnsName(name: string) {
+    return api.get<BlockchainApiLookupEnsName>({
+      path: `/v1/profile/account/${name}.wcn.id?projectId=${OptionsController.state.projectId}`
```

Comment:
> ..${name}.wcn.id..
As I remember, we removed `wcn.id`  - or did we just change it? Is this okay to keep? When we wanted to change it to something else, is this the only place we need to update?

Reply:
Can we move `wcn.id` to `core/src/utils/ConstantsUtil.ts`'s `CONSTANTS_UTIL` object so we could ref to it in both `BlockchainApiController` and `EnsController`?